server/service/integration_enterprise_test.go

```diff
@@ -0,0 +1,145 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/ptr"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestIntegrationsEnterprise(t *testing.T) {
+	testingSuite := new(integrationEnterpriseTestSuite)
+	testingSuite.s = &testingSuite.Suite
+	suite.Run(t, testingSuite)
+}
+
+type integrationEnterpriseTestSuite struct {
+	withServer
+	suite.Suite
+}
+
+func (s *integrationEnterpriseTestSuite) SetupSuite() {
+	s.withDS.SetupSuite("integrationEnterpriseTestSuite")
+
+	users, server := RunServerForTestsWithDS(
+		s.T(), s.ds, TestServerOpts{License: &fleet.LicenseInfo{Tier: fleet.TierBasic}})
+	s.server = server
+	s.users = users
+	s.token = s.getTestAdminToken()
+}
+
+func (s *integrationEnterpriseTestSuite) TestTeamSpecs() {
+	t := s.T()
+
+	// create a team through the service so it initializes the agent ops
+	teamName := t.Name() + "team1"
+	team := &fleet.Team{
+		Name:        teamName,
+		Description: "desc team1",
+	}
+
+	resp := s.Do("POST", "/api/v1/fleet/teams", team, http.StatusOK)
+	defer resp.Body.Close()
```

Comment:
If we passed the `*testing.T` to `s.Do`, it could automatically register the call to `resp.Body.Close()` in `t.Cleanup`, so it cannot be forgotten.